.zuul.yaml

```diff
@@ -0,0 +1,22 @@
+- job:
+    name: dnf-run-tests
+    description: Run DNF tests
+    roles:
+      - zuul: zuul-distro-jobs
+    requires: repo
+#    require-projects:
```

Comment:
This is commented for the moment, but we should add the SF github app to ci-dnf-stack and register it in the Zuul config. This statement will tells Zuul to checkout ci-dnf-stack into the job workspace (on the test node). Thank to it we will have access to the Dockerfile.